apps/web/src/components/tx-flow/flows/ManagerSigners/ReviewSigners.tsx

```diff
@@ -38,5 +41,27 @@ export function ReviewSigners(props: ReviewTransactionContentProps): ReactElemen
     createSafeTx().then(setSafeTx).catch(setSafeTxError)
   }, [data, safe, setSafeTx, setSafeTxError])
 
-  return <ReviewTransaction {...props} />
+  const addAddressBookEntryAndSubmit = () => {
```

Comment:
Nit: This function's name is misleading as it does not submit the form and adds multiple entries.

```suggestion
  const addAddressBookEntries = () => {
```